datadog/fwprovider/resource_datadog_team.go

```diff
@@ -75,6 +76,10 @@ func (r *teamResource) Schema(_ context.Context, _ resource.SchemaRequest, respo
 				Required:    true,
 				Description: "The name of the team.",
 			},
+			"avatar": schema.StringAttribute{
+				Required:    false,
+				Description: "The avatar of the team.",
```

Comment:
**Suggestion (non-blocking)**: maybe the description should specify that we expect a URL to an image asset?

Reply:
It looks like this is just an emoji string and not a URL:
<img width="792" alt="Screenshot 2024-09-10 at 3 51 52 PM" src="https://github.com/user-attachments/assets/0f8f0aa3-4aae-418b-a14c-e0d5b388d024">

Reply:
Oh, you are correct. Sorry about that. I forgot about that.